pkg/nuget/packagesprops/parse.go

```diff
@@ -0,0 +1,85 @@
+package packagesprops
+
+import (
+	"encoding/xml"
+	"strings"
+
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+type entry struct {
```

Comment:
`PackageReference` and `PackageVersion` => `package`
```suggestion
type package struct {
```

Reply:
package is reserved word, so I renamed it to pkg